src/main/java/com/artipie/settings/ArtipieAuthorization.java

```diff
@@ -0,0 +1,303 @@
+/*
+ * The MIT License (MIT) Copyright (c) 2020-2021 artipie.com
+ * https://github.com/artipie/artipie/LICENSE.txt
+ */
+package com.artipie.settings;
+
+import com.amihaiemil.eoyaml.YamlMapping;
+import com.amihaiemil.eoyaml.YamlNode;
+import com.amihaiemil.eoyaml.YamlSequence;
+import com.artipie.ArtipieException;
+import com.artipie.asto.Storage;
+import com.artipie.asto.blocking.BlockingStorage;
+import com.artipie.asto.factory.Config;
+import com.artipie.asto.factory.StoragesLoader;
+import com.artipie.auth.AuthFromEnv;
+import com.artipie.auth.AuthFromKeycloak;
+import com.artipie.auth.AuthFromStorage;
+import com.artipie.auth.GithubAuth;
+import com.artipie.http.auth.Authentication;
+import com.artipie.security.policy.PoliciesLoader;
+import com.artipie.security.policy.Policy;
+import com.artipie.security.policy.YamlPolicyConfig;
+import com.artipie.settings.cache.CachedUsers;
+import com.jcabi.log.Logger;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+import java.util.Optional;
+import java.util.function.Function;
+import org.keycloak.authorization.client.Configuration;
+
+/**
+ * Artipie authorization: authentication and permissions policy.
+ * @since 0.29
+ */
+public interface ArtipieAuthorization {
```

Comment:
The name `ArtipieAuthorization` doesn't reflect class's purpose

Reply:
Any suggestions?

Reply:
Maybe `ArtipieSecurity`